src/store/tipCalculatorApp/action.ts

```diff
@@ -0,0 +1,44 @@
+import { Dispatch, GetState } from '../types'
+import { ActionTypes, CALCULATOR_ACTIONS } from './types'
+
+export const SetBillAmount = (newAmount: number) => async (dispatch: Dispatch<ActionTypes>) => {
```

Comment:
all these actions should be in camelCase!

Reply:
Resolved in c0ae0a7